### PR TITLE
Various edits to caching docs

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: docs/source/caching/advanced-topics.mdx [hunk 1]
```diff
@@ -239,11 +239,11 @@ For details, see [The `fetchMore` function](../pagination/core-api/#the-fetchmor
 
 ### The `@connection` directive
```
Comment (StephenBarlow) [inline docs/source/caching/advanced-topics.mdx:240]: This section needs a more thorough pass but that requires more research, so just copyedits for now

File: docs/source/caching/cache-configuration.mdx [hunk 2]
```diff
@@ -4,10 +4,6 @@ title: Configuring the Apollo Client cache
 
 This article describes cache setup and configuration. To learn how to interact with cached data, see [Reading and writing data to the cache](./cache-interaction).
 
-## Installation
-
-In Apollo Client 3, the `InMemoryCache` class is provided by the `@apollo/client` package. No additional libraries are required.
```
Comment (StephenBarlow) [inline docs/source/caching/cache-configuration.mdx:9]: AC3's been around long enough that I think we can omit this difference from AC2 entirely

File: docs/source/caching/cache-field-behavior.mdx [hunk 3]
```diff
@@ -82,19 +87,19 @@ const cache = new InMemoryCache({
 If a field requires numerous parameters then each parameter must be wrapped in a variable that is then destructured and returned.
```
Comment (StephenBarlow) [inline docs/source/caching/cache-field-behavior.mdx:87]: I'm not entirely clear what this paragraph is trying to say in its current form, which prevented me from suggesting edits.

File: docs/source/caching/cache-field-behavior.mdx [hunk 4]
```diff
@@ -443,6 +412,33 @@ const cache = new InMemoryCache({
 
 Now that you've hidden the details behind a reusable abstraction, it no longer matters how complicated the implementation gets. This is liberating, because it allows you to improve your client-side business logic over time, while keeping related logic consistent across your entire application.
 
+### Defining a `merge` function at the type level
+
+In Apollo Client 3.3 and later, you can define a default `merge` function for a non-normalized object type. If you do, every field that _returns_ that type uses your default `merge` function unless it's overridden on a field-by-field basis.
```
Comment (StephenBarlow) [inline docs/source/caching/cache-field-behavior.mdx:417]: Not sure whether this override behavior works as I've currently described.

File: docs/source/caching/cache-field-behavior.mdx [hunk 5]
```diff
@@ -572,13 +568,36 @@ const cache = new InMemoryCache({
 });
 ```
 
+### Disabling `merge` functions
+
+In some cases, you might want to completely disable merge functions for certain fields. To do so, pass `merge: false` like so:
```
Comment (StephenBarlow) [inline docs/source/caching/cache-field-behavior.mdx:573]: Still need to come back to this with a little more clarification

File: docs/source/caching/cache-configuration.mdx (hunk 2 above)
Comment (jpvajda) [reply docs/source/caching/cache-configuration.mdx:9]: 👍

**Comment by bignimbus** [comment]
@StephenBarlow is there anything you need from the team here?

**Comment by StephenBarlow** [comment]
@bignimbus Ahh I'll need to review this PR to remind myself what it includes, which I probably can't do for a bit. Will follow up when possible!

**Comment by StephenBarlow** [comment]
Okay in updating and re-reviewing this PR, it looks entirely uncontroversial and just copy/structure related, so I'm going to go ahead and merge 👍

**Pull request merged**